kafka-producer/src/main/resources/config/base.conf

```diff
@@ -1,13 +1,32 @@
 {
```

Comment:
Comment for what each config does other that the obvious ones like kafka ones. Try to be as verbose as possible.